DEVELOPER_GUIDE.md

```diff
@@ -69,3 +69,14 @@ In the base OpenSearch Dashboards directory, run
 
   Runs end-to-end tests on a currently running OpenSearch Dashboards server. Defaults to run the tests on `localhost:5601`, although you can change this to run on any
   OpenSearch Dashboards server with the command `yarn cy:run --config baseUrl=<your-custom-URL>`
+
+
+### Formatting
```

Comment:
Can you sync with core dashboards folks about adding something there, and then plugins can copy/reference it in their respective repos? Just thinking of how to make it more standard across the dashboards plugins.

Reply:
Opensearch-Dashboards does have a section on Prettier (https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#prettier-and-linting). It is mainly mentioning the move to Prettier for code formatting and using Prettier and ESLint rules over any other rules. I'll reach out regarding CLI vs plugin and if they ever had any discrepancy issue.

Reply:
Oh cool, sounds good. If they have no preference I think it's fine to leave what you've added.

Reply:
synced up with core dashboards and they mentioned they also noticed discrepancies and they just rely on CLI for final checks pre-PR but don't really have a preference as long as code formatting uses prettier.